source/Buoy-Localization-Extensions/String.extension.st

```diff
@@ -0,0 +1,55 @@
+Extension { #name : 'String' }
+
+{ #category : '*Buoy-Localization-Extensions' }
+String >> asLanguageRange [
+
+  ^ LanguageRange fromString: self
+]
+
+{ #category : '*Buoy-Localization-Extensions' }
+String >> asLanguageTag [
+
+	^ LanguageTag fromString: self
+]
+
+{ #category : '*Buoy-Localization-Extensions' }
+String >> escaped [
+
```

Comment:
Create a separate class EscapingAlgorithm and put the code there, to prevent reimplementing this twice